unity-renderer/Assets/UIComponents/Scripts/Components/GridContainer/GridContainerComponentView.cs

```diff
@@ -403,10 +403,12 @@ public List<BaseComponentView> ExtractItems()
 
     public void RemoveItems()
     {
-        foreach (Transform child in transform)
+        List<BaseComponentView> itemsToDestroy = ExtractItems();
+        foreach (BaseComponentView itemToDestroy in itemsToDestroy)
         {
-            Destroy(child.gameObject);
+            DestroyImmediate(itemToDestroy.gameObject);
```

Comment:
Why can't we use the regular Destroy() ?

Reply:
The same as above, thanks!